pages/learn/accounts/enterprise-sso.md

```diff
@@ -9,31 +9,31 @@ __Note:__ This feature is currently available only on [Enterprise plans](https:/
 
 BalenaCloud Enterprise Single Sign-On (SSO) using SAML (Security Assertion Markup Language) allows organizations to manage user access and authentication through their existing Identity Providers (IdP). This integration enables users to use their corporate credentials to log in and access BalenaCloud services. By leveraging SAML, enterprises can simplify the login process, enhance security, streamline user management, and ensure compliance with their internal policies and procedures.
 
-Configuring an Identity Provider (IdP) as a login method requires a one-time setup process within both balenaCloud and the identity provider itself. Refer to our [IdP specific documentation][ms-saml] for detailed instructions on required configurations.
```

Comment:
I think we should leave (IdP) short hand as we later use that instead of writing out the full "Identity Provider". Its common practise to have the short hand in parenthesis the first time you introduce a term and then later in the section only describe it a with the short hand.

Reply:
We already mention the IdP short hand in the paragraph above this one. I think I should add back the shorthand here, but remove it from the first paragraph which is already loaded with terms and shorthands (and undo a commit that I just did which adds the short hand back to a subsequent paragraph).

Reply:
To me, it doesn't matter as long as it's clear and explicitly mentioned which it was and now is too.